nfpm.yaml

```diff
@@ -0,0 +1,47 @@
+name: pekwm
```

Comment:
As this is parameterized, worth naming it nfpm.yaml.in (as to not confuse with the dep?)